docs/doc/12-load-data/01-load/00-stage.md

```diff
@@ -159,7 +159,7 @@ Follow this tutorial to upload the sample file to an external stage and load dat
 
 ### Step 1. Create an External Stage
 
-1. Create an external stage with the [CREATE STAGE](../../14-sql-commands/00-ddl/40-stage/01-ddl-create-stage.md) command:
```

Comment:
have a try about changing "../../14-sql-commands/00-ddl/40-stage/04-ddl-list-stage.md" -into "/14-sql-commands/00-ddl/40-stage/04-ddl-list-stage.md" ?

so the doc framework can still check the links dependencies